src/plugins/base/base.js

```diff
@@ -11,16 +11,6 @@ module.exports = function () {
     // Heading defaults.
     var headings = config('theme.decanter.typography.headings');
 
-    // Input base styles.
-    var inputBase = {
-      display: 'block',
-      maxWidth: config('theme.decanter.base.inputMaxWidth'),
-      borderColor: config('theme.colors.foggy.dark'),
-      color: config('theme.colors.black.DEFAULT'),
-      fontSize: config('theme.decanter.base.inputFontSize'),
-      lineHeight: config('theme.lineHeight.snug'),
-    };
```

Comment:
Was this not being used?

Reply:
I decided to add .su-input-base as a TW "component" (since we have the different html base font compared to TW default), so removing it from the base.

Reply:
SG2M